pkgs/by-name/py/pybibget/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, python3
+, fetchPypi
+}:
+
+let
+  pname = "pybibget";
+  version = "0.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1n4as3kdsgrcmxxyjz9klm0v0pb24rsa1b7r43j7wmnfsir8i81k";
+  };
+
+in
+python3.pkgs.buildPythonApplication  {
+  inherit pname version src;
+  pyproject = true;
+
+  propagatedBuildInputs = with python3.pkgs; [
+    lxml
+    httpx
+    appdirs
+    aiolimiter
+    pybtex
+    pylatexenc
+    numpy
+    networkx
+    requests
+  ];
+
+  # Assuming you have tests and they can be run by pytest
+  # If not, you might want to set `doCheck = false;`
+  # or adjust the checkPhase to properly run the package's tests.
+  doCheck = false;
+  #checkInputs = [ pytestCheckHook ];
+  #pythonImportsCheck = [ "pybibget" ];
```

Comment:
These comments can be cleaned up a bit, if we're disabling checks, just add a comment explaining why